packages/frontend-api/src/Model/Resolver/Category/CategoryResolverMap.php

```diff
@@ -0,0 +1,45 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrontendApiBundle\Model\Resolver\Category;
+
+use Overblog\GraphQLBundle\Resolver\ResolverMap;
+use Shopsys\FrameworkBundle\Component\Domain\Domain;
+use Shopsys\FrameworkBundle\Model\Category\Category;
+
+class CategoryResolverMap extends ResolverMap
+{
+    /**
+     * @var \Shopsys\FrameworkBundle\Component\Domain\Domain
+     */
+    protected $domain;
+
+    /**
+     * @param \Shopsys\FrameworkBundle\Component\Domain\Domain $domain
+     */
+    public function __construct(Domain $domain)
+    {
+        $this->domain = $domain;
+    }
+
+    /**
+     * @return array
+     */
+    protected function map(): array
+    {
+        return [
+            'Category' => [
+                'seoH1' => function (Category $category) {
```

Comment:
Can you please consult with PO, whether we want to return the default value on null.

In administration you see a placeholder with default value (from general eshop seo settings).

Or there should be a query to return general seo settings.

Reply:
I am more inclined to returning "null" value. It's up to developer of consumer App to decide if there should be default value and what it shoulld be.
...and yes, there should be query for returning "domain" data (discussed elsewhere), including generic SEO data.